### ISSUE TITLE
Create Input Message Component

### ISSUE THREAD

**Issue by skuhlmann** [HausDAO/daohaus-monorepo#26]
# Overview

Create a reusable basic input message component.

## Details

Name: `<InputMessage />``

### Spec questions

- Dependent on some Typography/icons

#### Variants

https://www.figma.com/file/R1a1bwODnzCHHHw0EJALAU/DAOhaus-v3-Pattern-Library-(In-Progress)?node-id=92%3A8758

- warning w/ icon
- valid value w/ icon
- error w/ icon

#### Props
- messageText
- messageType (warn, error, valid)

## Resources/Design

PL Page: https://www.figma.com/file/R1a1bwODnzCHHHw0EJALAU/DAOhaus-v3-Pattern-Library-(In-Progress)?node-id=92%3A8758

## Deliverable
- Code/PR
- Storybook deployment

**Comment by earth2travis** [comment]
Blocked by: https://github.com/HausDAO/daohaus-monorepo/issues/36

**Comment by jordanlesich** [comment]
Changes to the spec. 

This was written a while back and we've made some changes since then. Most of the logic now happens in the FieldWrapper component, also we have new considerations like theming and dark/light mode. 

this spec will be more about building a bunch of the input primitives. The field wrapper spec will handle their logic and states. 

Here's what this spec should include
- [ ] input atom (with standard state changes)
- [ ] input atom border state change (error, warning, etc)
- [ ] add on variants (excluding button)
- [ ] input consumes color values from theme + handles light/dark shifts

The FieldWrapper is what composes the helperText.

 I'll build message text as a separate story and include it on the PR for this issue, but won't integrate it as a macro until
